packages/graphql-language-service-server/src/MessageProcessor.ts

```diff
@@ -8,8 +8,9 @@
  */
 
 import mkdirp from 'mkdirp';
+// can't use node:fs since we mock in jest `fs` module
```

Comment:
Would it work if we replace `jest.mock('fs')` with `jest.mock('node:fs')` in the test file? As an alternative, we could remove mock for the whole module and just spy on `readFileSync` in the individual tests.

Reply:
I tried `jest.mock('node:fs')` but for some reason it not worked 🥲

Reply:
@thomasheyenbrock you were right, I forget to modify `jest.requireMock('fs')` as well